src/plugins/intel_cpu/tests/functional/subgraph_tests/src/classes/conv_maxpool_activ.hpp

```diff
@@ -0,0 +1,23 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "test_utils/fusing_test_utils.hpp"
+#include "ov_models/builders.hpp"
+
+using namespace ngraph;
+using namespace InferenceEngine;
+using namespace CPUTestUtils;
+
+namespace SubgraphTestsDefinitions {
+
+using ConvPoolActivTestParams = fusingSpecificParams;
+
+class ConvPoolActivTest : public testing::WithParamInterface<ConvPoolActivTestParams>, public CpuTestWithFusing,
+                          virtual public LayerTestsUtils::LayerTestsCommon {
```

Comment:
please, inherit from `SubgraphBaseTest` instead of `LayerTestsCommon`

Reply:
Done